src/components/BookList.js

```diff
@@ -11,43 +13,52 @@ const BookList = () => {
     dispatch(getBooks());
   }, [dispatch]);
 
+  const progress = Math.round(Math.random() * (100 - 0) + 0);
+  const chapter = Math.round(Math.random() * (20 - 0) + 0);
+
   return (
-    <>
+    <div className="card card-style mx-auto my-2">
       {bookStore.map((book) => (
-        <div key={book.id} className="d-flex flex-row justify-content-center border">
+        <div key={book.id} className="d-flex flex-row align-items-center justify-content-between my-2">
           <div className="d-flex flex-column px-3">
-            <h3>Category</h3>
-            <h1>
+            <h5 className="text-secondary">{book.category}</h5>
+            <h3>
               {book.title}
-            </h1>
-            <span>
-              {book.author}
-            </span>
+            </h3>
+            <h5 className="author text-primary fw-light">{book.author}</h5>
             <div>
-              <ul>
-                <li>Comments</li>
+              <ul className="d-flex flex-row text-decoration-none navbar-nav">
+                <li className="text-primary fw-light me-2">Comments</li>
+                <li> | </li>
                 <li>
                   <RemoveBook id={book.id} book={book} />
                 </li>
-                <li>Edit</li>
+                <li> | </li>
+                <li className="text-primary fw-light mx-2">Edit</li>
               </ul>
             </div>
           </div>
-          <div className="px-3">
-            <div>Animated Circle</div>
-            <div>XX%</div>
-            <h4>Completed</h4>
+          <div className="d-flex flex-row px-3">
+            <div className="m-2"><CircularProgress color="primary" variant="determinate" value={progress} /></div>
+            <div className="flex-column">
+              <h2>
+                {progress}
+                %
+              </h2>
+              <h5 className="completed">Completed</h5>
+            </div>
```

Comment:
- [ ] Each two book should have some space between them, I would suggest copying the font styles from the design for the `book` component so it matches it (`font-size` and `font-weight`).
- [ ] The vertical separator `|` between `progress` and `chapter info` is missing please add it so it matches the design.

Zeplin Design
![image](https://user-images.githubusercontent.com/33767951/142711938-fb734295-22f0-4eac-9f5a-3ad26a054da2.png)

Your Styling
![image](https://user-images.githubusercontent.com/33767951/142711861-7e7c1cf7-d9de-4d30-9e9e-f7eebed39b80.png)